packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/javascript/line.scope

```diff
@@ -0,0 +1,117 @@
+
+aaa
+
+---
+
+[#1 Content]
+ 
+[]
+ aaa
+ 
+
+[#1 Removal]
+[
+ 
+ aaa
+ ]
```

Comment:
this is nice, tho things def do get a bit confusing with empty lines. I do wonder if we want to move to using a prefix for content lines so it's crystal clear which lines are content. We could even use a prefix and a suffix, so then empty lines are very clear, because they'd be eg `><`

Reply:
We could use pipes along the left and right hand side to create a box? Kinda like mark down tables?

Reply:
Something like this? 
```
|[    |
|     |
| aaa |
| ]   |
```

Or without the padding. Note that the starting bracket is now part of the wall.

```
[   |
|   |
|aaa|
|]  |
```

Reply:
I like the pipes, but

- they should hug the content on the right rather than lining up, otherwise they're not giving us any information
- there should be no pipe on the annotation lines, just content lines

Reply:
Got it. Like this?

```
[
||
|aaa|
 ]
```

Reply:
yep that's exactly what I'm proposing. We could tweak from there, but I do think something like that makes whitespace very clear

Reply:
I'm not sure if that would be easier to read.

Reply:
discussed in meet-up:

- Only add the marker at the end of the line
- Use something like `◂` as the marker
- If it's too noisy, don't include the marker if the line ends with a nonwhitespace char (ie you'd only get it for empty lines or lines with trailing whitespace)

Reply:
This is now implemented

Reply:
@josharian better?

Reply:
I couldn't figure out how to understand these from just looking at the files, which would be the best case goal. I'm probably just missing something obvious, but why aren't we using `^` in these cases?

Reply:
It's an empty line. The carets are on the character, but the squares are outside of the range and therefore works.